src/AasxServerBlazor/Pages/Db.razor

```diff
@@ -1,905 +1,803 @@
+                            <br />
+                        }
+                        <br />
+                    }
+                    <span>@("Count: " + data.Count)</span>
+                    <br />
+                }
+
+                if (url.EndsWith("/sm"))
+                {
+                    List<SMSet> data = PageDB.GetPageSMData(size, searchLower, aasxid, aasid, smid);
+                    <span>
+                        &nbsp&nbsp@("SM#: " + data.Count + "/" + db.SMSets.Count() + " size=" + size + " search=" + search + " aasxid=" + aasxid + " aasid=" + aasid + " smid=" + smid)
+                    </span>
+                    <br />
+                    <br />
+                    @if (size > 1000 && data.Count > 1000)
+                    {
+                        <b><span style="color:red;">First 1000 as TABLE</span></b>
+                        <br />
+                    }
+                    <table class="table table-bordered table-sm" style="width:100%;word-wrap:break-word;word-break:break-all;">
+                        <thead>
+                            <tr>
+                                <th style="word-break:keep-all">AASX#</th>
+                                <th style="word-break:keep-all">AAS#</th>
+                                <th style="word-break:keep-all">SM#</th>
+                                <th style="word-break:keep-all">Id</th>
+                                <th style="word-break:keep-all">IdShort</th>
+                                <th style="word-break:keep-all">SemanticId</th>
+                                <th style="word-break:keep-all">Links</th>
+                                <th style="word-break:keep-all">API</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @for (int i = 0; i < data.Count && i < 1000; i++)
+                            {
+                                var smDB = data[i];
+                                string submodel64 = Base64UrlEncoder.Encode((smDB.Identifier));
+                                string link = AasxServer.Program.externalBlazor + "/submodels/" + submodel64;
+                                int smeCount = db.SMESets.Where(sme => sme.SMId == smDB.Id).Count();
+
+                                <tr>
+                                    <td style="word-break:keep-all">@smDB.AASXId</td>
+                                    <td style="word-break:keep-all">@smDB.AASId</td>
+                                    <td style="word-break:keep-all"><b>@smDB.Id</b></td>
+                                    <td>@smDB.Identifier</td>
+                                    <td>@smDB.IdShort</td>
+                                    <td>@smDB.SemanticId</td>
+                                    <td style="word-break:keep-all">
+                                        <a href="@(shorturl + "/aasx?aasxid=" + smDB.AASXId)" target="_blank">aasx </a>

```

Comment:
Please use string concatenation, as this is better to read and it is easier to spot changes.
```suggestion
<a href="@($"{shorturl}/aasx?aasxid={smDB.AASXId})" target="_blank">aasx </a>
```